apm-lambda-extension/extension/apm_server_transport.go

```diff
@@ -232,6 +232,13 @@ func (transport *ApmServerTransport) SetApmServerTransportState(ctx context.Cont
 
 // ComputeGracePeriod https://github.com/elastic/apm/blob/main/specs/agents/transport.md#transport-errors
 func (transport *ApmServerTransport) computeGracePeriod() time.Duration {
+	// If reconnectionCount is 0, returns a random number in an interval.
+	// The grace period for the first reconnection count was 0 but that
+	// leads to collisions with multiple environment.
+	if transport.reconnectionCount == 0 {
+		gracePeriod := rand.Float64() * 5
```

Comment:
Should seed the rand library via `rand.Seed()` or create a new `*rand.Rand` and use it in ApmServerTransport. WDYT?

Reply:
Sorry, I missed this comment :bow: 

That's a great idea! I've seeded the rand library :+1: